README.md

```diff
@@ -532,6 +532,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Brian Buddhadasa](https://github.com/brianbud)
   - [Brian Kiarie Mwaniki](https://github.com/BrianKiarieMwaniki)
   - [Brian Nduhiu](https://github.com/Brian-Nduhiu)
+  - [Brian Kepha](https://github.com/AngelofVerdant)
```

Comment:
Please add your name in line 533, after Brian Buddhadasa.
Let us know if you need any help with that! 🙂

Reply:
Okay, will do